File: Linguagem C/while/ex_008.c
//8) Faça um programa que leia dois inteiros N e X e mostre a soma dos múltiplos de X menores que N. Não
//deixe o usuário inserir valores negativos.
#include <stdio.h>
int main (){
int N, X, soma=0, i=1;
    do{
    printf("Digite um numero: ");
    scanf("%d", &N);
    } while (N < 0);

    do{
    printf("Digite um numero: ");
    scanf("%d", &X);
    } while (X < 0);
    
    while (i < N){
        if( i % X == 0){
        soma+=i;
        }
        i++;
    }
    printf("%d ", soma);
}

File: Python/005_Lista_While/Ex_008.py
"""
8) Faça um programa que leia dois inteiros N e X e mostre a soma dos múltiplos de X menores que N. Não 
deixe o usuário inserir valores negativos.
"""

n = int(input("Digite um valor para n: "))
x = int(input("Digite um valor para x: "))
soma = 0

for num in range(1, n, 1):
    if num  %  x == 0:
        soma += num
print(soma)
